apps/desktop/src/main/lib/agent-setup/templates/gemini-hook.template.ps1

```
@@ -0,0 +1,58 @@
+
+$ErrorActionPreference = 'Continue'
+
+function Get-JsonStringValue([string]$Text, [string]$Key) {
+	if (-not $Text) { return '' }
+	$pattern = '"' + [regex]::Escape($Key) + '"\s*:\s*"([^"]*)"'
+	$match = [regex]::Match($Text, $pattern)
+	if ($match.Success) { return $match.Groups[1].Value }
+	return ''
+}
+
+$Input = [Console]::In.ReadToEnd()
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'Input' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)